src/lib/db/context-field-store.ts

```diff
@@ -16,7 +17,16 @@ const COLUMNS = [
 ];
 const TABLE = 'context_fields';
 
-const mapRow: (object) => IContextField = (row) => ({
+type ContextFieldDB = {
+    name: string;
+    description: string;
+    stickiness: boolean;
+    sort_order: number;
+    legal_values: ILegalValue[];
+    created_at: Date;
+};
+
+const mapRow = (row: ContextFieldDB): IContextField => ({
```

Comment:
Apparently, we hadn't typed this at all before. I've added a type that describes the mapping we do.